webhook_to_fedora_messaging/config.py

```diff
@@ -4,13 +4,15 @@
 
 from functools import cache
 from pathlib import Path
-
+import os
 from pydantic import BaseModel, ConfigDict, DirectoryPath
 from pydantic_settings import BaseSettings
 
 
 DEFAULT_CONFIG_FILE = _config_file = "/etc/webhook-to-fedora-messaging/webhook-to-fedora-messaging.cfg"
```

Comment:
I see that you are using the INI-styled configuration here. Could you please include a default configuration with placeholder values for folks to get started with?

Reply:
Also, @abompard should we stick to an INI-styled configuration or use a more profound TOML-styled configuration? I am fine with either but I wanted to hear your opinion on this.

Reply:
We shouldn't use pydantic anyway, because Flask has its own configuration system.

Reply:
Yeah, I got bamboozled seeing all the Pydantic stuff here and assumed that it was used here.